splunk-otel-android-volley/src/main/java/com/splunk/rum/VolleyHttpClientAttributesGetter.java

```diff
@@ -116,4 +127,9 @@ static List<String> headersToList(List<Header> headers, String name) {
         }
         return headersList;
     }
+
+    @Override
+    public String getTransport(RequestWrapper requestWrapper, HttpResponse httpResponse) {
+        return SemanticAttributes.NetTransportValues.IP_TCP;
+    }
```

Comment:
Can you also add a `getNetworkTransport()` method? The `getTransport()` one will be removed in 2.0
```suggestion
    }
    
    @Override
    public String getNetworkTransport(RequestWrapper requestWrapper, HttpResponse httpResponse) {
        return "tcp";
    }
```

Reply:
Cool sorry I missed that one! Thanks!